mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -220,7 +221,10 @@ func resourceMongoDBAtlasProjectUpdate(ctx context.Context, d *schema.ResourceDa
 
 			_, err := conn.Teams.RemoveTeamFromProject(ctx, projectID, teamID)
 			if err != nil {
-				return diag.Errorf("error removing team(%s) from the project(%s): %s", teamID, projectID, err)
+				var target *matlas.ErrorResponse
+				if errors.As(err, &target) && target.ErrorCode != "USER_UNAUTHORIZED" {
+					return diag.Errorf("error removing team(%s) from the project(%s): %s", teamID, projectID, err)
+				}
```

Comment:
[nit] I wouldn’t silently ignore the error `USER_UNAUTHORIZED` but at least print a warning so that the user is aware that the team removal operation failed because they don't have the correct permission.